contract/package.json

```diff
@@ -11,26 +11,32 @@
     "docker:make": "docker compose exec agd make -C /workspace/contract",
     "make:help": "make list",
     "start": "yarn docker:make clean start-contract print-key",
-    "build": "exit 0",
+    "build": "node_modules/agoric/bin/agoric run scripts/build-game1-start.js",
     "test": "ava --verbose",
-    "lint": "eslint '**/*.{js,ts}'",
-    "lint-fix": "eslint --fix '**/*.{js,ts}'",
-    "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.js'",
-    "lint-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.js'",
-    "typecheck": "tsc"
+    "lint": "yarn lint:eslint # && yarn lint:types",
+    "lint:eslint": "eslint '**/*.{js,ts}'",
+    "lint:fix": "eslint --fix '**/*.{js,ts}'",
+    "lint:types": "tsc"
+  },
+  "dependencies": {
+    "@agoric/ertp": "0.16.3-u13.0",
+    "@agoric/zoe": "0.26.3-u13.0",
```

Comment:
Have you checked that the resulting bundle is a reasonable size? that we have just 1 version of each endo package?